website/docs/docs/collaborate/govern/model-access.md

```diff
@@ -9,27 +9,54 @@ description: "Define model access with group capabilities"
 This functionality is new in v1.5.
 :::
 
+:::info "Model access" is not "User access"
+
+**Model groups and access** and **user groups and access** mean two different things. "User groups and access" is a specific term used in dbt Cloud to manage permissions. Refer to [User access](https://github.com/docs/cloud/manage-access/about-user-accessl) for more information.
```

Comment:
```suggestion
**Model groups and access** and **user groups and access** mean two different things. "User groups and access" is a specific term used in dbt Cloud to manage permissions. Refer to [User access](/docs/cloud/manage-access/about-user-access) for more information.
```

Reply:
whoops!! yes, totally ok, thank you for fixing 🙏